src/library/Hooks/usePoolCommission/index.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright 2023 @paritytech/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { useBondedPools } from 'contexts/Pools/BondedPools';
+import { usePoolsConfig } from 'contexts/Pools/PoolsConfig';
+
+export const usePoolCommission = (id: number) => {
```

Comment:
```suggestion
export const usePoolCommission = (id: number): number => {
```

Reply:
I think you mean getCurrentCommission(), I have edited to number | undefined to avoid the type complaint